R/Utility.R

```diff
@@ -123,3 +123,17 @@ guessPythonExePathEnvironmentVariables <- function(){
     pythonExePaths
 }
 
+getPandasDataFrame <- function(key){
+  pyExec(sprintf("x = %s.to_dict(orient='list')", key))
+  return( as.data.frame(pyGet("x"), optional=TRUE, stringsAsFactors=FALSE) )
+}
+
+getOrderedDict <- function(key){
```

Comment:
need a test in which a null value is in the OrderedDict, but NOT at the end.  Is there a case in which a *key* is null?